sql/core/src/test/gen-java/org/apache/spark/sql/execution/datasources/parquet/test/avro/AvroArrayOfArray.java

```diff
@@ -129,6 +129,7 @@ public boolean hasIntArraysColumn() {
     }
 
     @Override
+    @SuppressWarnings(value="unchecked")
```

Comment:
Hi @srowen By default the code generated in avro use @SuppressWarnings(value="unchecked"), e.g.,
https://github.com/apache/spark/blob/master/sql/core/src/test/gen-java/org/apache/spark/sql/execution/datasources/parquet/test/avro/AvroArrayOfArray.java#L37

I can do a search and replace to remove all "value=". Would you prefer to do it in this PR or separately?